src/sentry/tasks/post_process.py

```diff
@@ -699,12 +713,26 @@ def process_snoozes(job: PostProcessJob) -> None:
                 "user_count": snooze.user_count,
                 "user_window": snooze.user_window,
             }
-            if features.has("organizations:issue-states", group.organization):
-                add_group_to_inbox(group, GroupInboxReason.ONGOING, snooze_details)
-                record_group_history(group, GroupHistoryStatus.ONGOING)
+            if features.has("organizations:escalating-issue", group.organization):
+                add_group_to_inbox(group, GroupInboxReason.ESCALATING, snooze_details)
+                record_group_history(group, GroupHistoryStatus.ESCALATING)
+                group.substatus = GroupSubStatus.ESCALATING
+                group.status = GroupStatus.UNRESOLVED
+                group.save(update_fields=["status", "substatus"])
+                analytics.record(
+                    "issue.escalating",
+                    organization_id=group.project.organization.id,
+                    project_id=group.project.id,
+                    group_id=group.id,
+                )
+
             else:
                 add_group_to_inbox(group, GroupInboxReason.UNIGNORED, snooze_details)
                 record_group_history(group, GroupHistoryStatus.UNIGNORED)
```

Comment:
If we plan on GA for issue states before escalating, we need to support a non-escalating case here with `GroupInboxReason.ONGOING`

Reply:
That makes sense! I still think we may need something like this since 
1) we're releasing escalating changes after issue states and 
2) we're getting rid of unignored in issue states

```
if escalating:
  your_new_logic_here
elif issue_states:
  # use the ongoing label instead of unignored
else:
  # current logic